### ISSUE TITLE
accessibility issue: telegram  desktop and screen reader

### ISSUE THREAD

**Issue by lstar4ever** [telegramdesktop/tdesktop#2215]
### Steps to reproduce
1.Downloaded portable version for windows  
2. run the exe file 
3. cannot interactwith screen reader NVDA  

### Expected behaviour
Tell us what should happen
should be able to navigate using keyboard   
e.g. should be able to move from one item to the other by tab
 

### Actual behaviour
Tell us what happens instead
unable to interact with screen reader by keyboard, e.g. tab have no respnce
### Configuration
**Operating system:**
Win 7 pro 
**Version of Telegram Desktop:**
not sure

**Comment by auchri** [comment]
Duplicate of #476
